src/Kuzzle.php

```diff
@@ -101,6 +105,12 @@ public function __construct($url, array $options = [])
             $this->requestHandler = new CurlRequest();
         }
 
+        if (array_key_exists('port', $options)) {
+            $this->port = $options['port'];
+        }
+
+
+        $this->url = 'http://' . $host . ':' . $this->port;
```

Comment:
should also set the protocol as an option, because we could (and should, in a production environment), put kuzzle behind a SSL proxy to reach it through https:// urls.